components/social-media-icon/social-media.constant.js

```diff
@@ -0,0 +1,22 @@
+export const IconMapper = {
+  twitter: {
+    alt: 'twitter',
+    src: 'icons/twitter.png',
+    href: '//twitter.com/'
```

Comment:
`//`?

Reply:
// used for overriding the local / test environment base URL

Reply:
ok